draft-ietf-rtcweb-jsep.xml

```diff
@@ -741,66 +741,80 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
-          that when considering a MediaStreamTrack that is producing
-          rotated video, the unrotated resolution MUST be used. This is
-          required regardless of whether the receiver supports
-          performing receive-side rotation (e.g., through CVO
-          <xref target="TS26.114" />), as it significantly simplifies
-          the matching logic.</t>
-
-          <t>For the purposes of resolution negotiation, only size
-          limits are considered. Any other values, e.g. picture or
-          sample aspect ratio, MUST be ignored.</t>
-
-          <t>When communicating with a non-JSEP endpoint, multiple
-          relevant "a=imageattr recv" attributes may be present in a
-          received m= section. If this occurs, attributes other than
-          the one with the highest "q=" value MUST be ignored. If
-          multiple attributes have the same "q=" value, those that
-          appear after the first such attribute in the m= section MUST
-          be ignored.</t>
-
-          <t>If an "a=imageattr recv" attribute references a different
-          video payload type than what has been selected for sending
-          the MediaStreamTrack, it MUST be ignored.</t>
-
-          <t>If the original resolution matches the size limits in the
-          attribute, the track MUST be transmitted untouched.</t>
-
-          <t>If the original resolution exceeds the size limits in the
-          attribute, the sender SHOULD apply downscaling to the output
-          of the MediaStreamTrack in order to satisfy the limits.
-          Downscaling MUST NOT change the track aspect ratio.</t>
-
-          <t>If the original resolution is less than the size limits in
-          the attribute, upscaling is needed, but this may not be
-          appropriate in all cases. To address this concern, the
-          application can set an upscaling policy for each sent track.
-          For this case, if upscaling is permitted by policy, the
-          sender SHOULD apply upscaling in order to provide the desired
-          resolution. Otherwise, the sender MUST NOT apply upscaling.
-          The sender SHOULD NOT upscale in other cases, even if the
-          policy permits it. Upscaling MUST NOT change the track aspect
-          ratio.</t>
-
-          <t>If there is no appropriate and permitted scaling mechanism
-          that allows the received size limits to be satisfied, the
-          sender MUST NOT transmit the track.</t>
-
-          <t>If the attribute includes a "sar=" (sample aspect ratio)
-          value set to something other than "1.0", indicating the
-          receiver wants to receive non-square pixels, this cannot be
-          satisfied and the sender MUST NOT transmit the track.</t>
-
-          <t>In the special case of receiving a maximum resolution of
-          [0, 0], as described above, the sender MUST NOT transmit the
-          track.</t>
+          <t>This specification prescribes more specific behavior.
```

Comment:
I chose not to use a colon because the rest of this paragraph is in sentence rather than list form.